kotlin-ktor/src/main/java/com/baeldung/graphql/client/GraphQLService.kt

```diff
@@ -0,0 +1,61 @@
+package com.baeldung.graphql.client
+
+import com.baeldung.graphql.client.generated.ConferenceByIdQuery
+import com.baeldung.graphql.client.generated.ObjectByIdQuery
+import com.baeldung.graphql.client.generated.SaveOrCreateAttendeeMutation
+import com.baeldung.graphql.client.generated.SaveOrCreateConferenceMutation
+import com.baeldung.graphql.client.generated.conferencebyidquery.Conference
+import com.baeldung.graphql.client.generated.objectbyidquery.ObjectWithId
+import com.baeldung.graphql.client.generated.saveorcreateattendeemutation.Attendee
+import com.expediagroup.graphql.client.jackson.GraphQLClientJacksonSerializer
+import com.expediagroup.graphql.client.ktor.GraphQLKtorClient
+import java.net.URI
+
+class GraphQLService(url: String) {
```

Comment:
For clarity, can we rename this to `GraphQLClient` or similar, to emphasise it contains the client side code. This would then match the corresponding client test name.

Reply:
✅ I thought this might cause confusion with the expedia group client, but they do have the added 'Ktor' in it, so I think it's alright